.github/pull.yml

```diff
@@ -0,0 +1,5 @@
+version: "1"
+rules:
+  - base: master
+    upstream: alphagov:master
+    mergeMethod: merge
```

Comment:
Could we have a new line :)

Reply:
Oh go on then.

Reply:
I assume that's where you mean...

Reply:
no I meant at the end of the file

Reply:
Ooh, I see. I did think it was an odd style. :)

Reply:
done :)